doc/tutorials/inter-vm_communication.rst

```diff
@@ -0,0 +1,490 @@
+.. _inter-vm_communication:
+
+ACRN Inter-VM Communication
+##############################################
+
+ACRN supports three kinds of Inter-VM communication – Inter-VM vUART,
+Inter-VM network communication, and Inter-VM shared memory device (ivshmem).
+Each communication method has its pros and cons, as described below.
+
+Inter-VM vUART
+**************
+
+Inter-VM vUART communication is based on the vUART implementation. It is
+used to transfer data between two VMs at low speed(< 120kbps). (Refer to :ref:`vuart_virtualization`
+and :ref:`vuart_config`).
+
+:numref:`Inter-VM vUART communication` shows the Inter-VM vUART communication overview:
+
+.. figure:: images/Inter-VM_vUART_communocation_overview.png
+   :align: center
+   :name: Inter-VM vUART communication
+
+   Inter-VM vUART communication
+
+- Pros:
+   - POSIX APIs; development-friendly (easily used programmatically
+     by using standard POSIX APIs).
+   - UART driver is available in most VMs.
```

Comment:
I think it would more accurate that the "UART drivers are available in most Operating Systems", wouldn't it?

Reply:
yes, Thanks, has been fixed.